src/components/DetailedView/DetailedView.jsx

```diff
@@ -0,0 +1,7 @@
+export default function DetailedView() {
```

Comment:
Component to use at future as container of detailed weather data for a specific day